integration_tests/models/schema_tests/schema.yml

```diff
@@ -274,9 +274,9 @@ models:
           - dbt_expectations.expect_column_distinct_values_to_contain_set:
               value_set: ['a','b']
               quote_values: true
-
-          - dbt_expectations.expect_column_value_lengths_to_equal   :
+          - dbt_expectations.expect_column_value_lengths_to_equal:
               value: 1
+          - dbt_expectations.expect_column_values_to_have_consistent_casing
```

Comment:
Can we also please test the `display_inconsistent_columns: true` case since that's very different SQL?

Reply:
Great, added as a new testing instance